src/index.js

```diff
@@ -624,15 +634,27 @@ export default function Vidy(opts) {
 		actions._clip = vlink._clipid; // for likes & comments
 		video._uuid = vlink._uuid; // for share url
 
-		// Render "liked" visual for this Embed
-		toggle(actions, styles['liked'], !!vlink._user_likes);
+		// Render "reaction" visual for this Embed
+		let hasReaction = !!vlink._user_likes || !!vlink._user_angers || !!vlink._user_surprises || !!vlink._user_laughs;
+		// for reference to current reaction
+		toggle(actions, styles['like'], !!vlink._user_likes);
+		toggle(actions, styles['anger'], !!vlink._user_angers);
+		toggle(actions, styles['surprise'], !!vlink._user_surprises);
+		toggle(actions, styles['laugh'], !!vlink._user_laughs);
+		// update UI
+		toggle(actions, styles['poll'], hasReaction);
+		toggle(actions.childNodes[0], styles['active'], vlink._user_likes);
```

Comment:
Don't we have named references? 

```suggestion
		toggle(actions._like, styles['active'], vlink._user_likes);
```